airbyte-integrations/connectors/source-github/source_github/spec.json

```diff
@@ -4,25 +4,38 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Github Source Spec",
     "type": "object",
-    "required": ["access_token", "repository", "start_date"],
+    "required": ["access_token", "start_date"],
     "additionalProperties": false,
     "properties": {
       "access_token": {
         "type": "string",
         "description": "Log into Github and then generate a <a href=\"https://github.com/settings/tokens\"> personal access token</a>.",
         "airbyte_secret": true
       },
+      "organization": {
```

Comment:
it's a little confusing the interaction between organization and repositories. 

This should instead be a oneOf where the user can choose to sync either organizations or specific repositories. See https://docs.airbyte.io/connector-development/connector-specification-reference#using-oneofs 

Backwards compatibility will be an interesting challenge though.

Reply:
I changed spec.json file as you asked and it really looks much more user-friendly. 
But I still can't think of a way in which we will maintain backward compatibility with the previous version.

Reply:
Ok, let's instead make both organizations and repositories available as input fields, and take the inputs from both of them (instead of just one). I think this is probably a better UX anyways since it's more flexible. 

We should verify that at least one of them is set though in the `check` method.